.clippy.toml

```diff
@@ -6,16 +6,17 @@ disallowed-methods = [
     "std::env::set_var",
     "std::env::remove_var",
 
-    # Use parking_lot instead.
-    "std::sync::Mutex",
-    "std::sync::RwLock",
-
-    # Use tokio::time::Instant instead.
-    "std::time::Instant::now",
-
     # Avoid instances of https://github.com/rust-lang/rust/issues/86470 until tokio/tokio#4461 is
     # available.
     "tokio::time::Instant::duration_since",
     "tokio::time::Instant::elapsed",
     # Clippy doesn't let us ban tokio::time::Instant::sub, but it knows what it did.
 ]
+disallowed-types = [
```

Comment:
i think we might also need to add `#![deny(clippy::disallowed_types)]` to every crate in the project. based on a quick grep, there are definitely a few uses of `std::sync` locks in the codebase, so i'm surprised this isn't catching them...

most uses are in tests, but there's at least one `std::sync::RwLock` in `linkerd/stack/src/fail_on_error.rs` according to grep.

Reply:
thanks. fixed